core/audits/metrics/max-potential-fid.js

```diff
@@ -55,43 +63,51 @@ class MaxPotentialFID extends Audit {
    * debugdata details.
    * @param {LH.Artifacts.ProcessedTrace} processedTrace
    * @param {LH.Artifacts.ProcessedNavigation} processedNavigation
-   * @return {{type: 'debugdata', observedMaxDurationLoaf: LH.TraceEvent, observedMaxBlockingLoaf: LH.TraceEvent}|undefined}
+   * @return {LoafDebugDetails|undefined}
    */
   static getLongAnimationFrameDetails(processedTrace, processedNavigation) {
-    const {firstContentfulPaint} = processedNavigation.timestamps;
-    const loafs = processedTrace.mainThreadEvents.filter(evt => {
-      return evt.name === 'LongAnimationFrame' &&
-          evt.ph === 'b' &&
-          evt.ts >= firstContentfulPaint;
+    const {firstContentfulPaint, timeOrigin} = processedNavigation.timestamps;
+
+    const loafEvents = processedTrace.mainThreadEvents.filter(evt => {
+      return evt.name === 'LongAnimationFrame' && evt.ph === 'b';
     });
+    if (loafEvents.length === 0) return;
```

Comment:
This case won't really hurt anything without this check, so is mostly for aesthetics (prevent empty `{type: 'debugdata', observedLoafs: []}` hanging around in LHRs)